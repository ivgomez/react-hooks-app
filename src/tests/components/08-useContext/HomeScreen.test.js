import "@testing-library/jest-dom";
import { mount } from "enzyme";
import React from "react";
import { HomeScreen } from "../../../components/08-useContext/HomeScreen";
import { UserContext } from "../../../components/08-useContext/UserContext";

describe("Test on <HomeScreen/>", () => {
  const user = {
    name: "papaleta",
    email: "pp@mail.com",
  };
  const wrapper = mount(
    <UserContext.Provider value={{ user }}>
      <HomeScreen />
    </UserContext.Provider>
  );
  test("should rendered right ", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
