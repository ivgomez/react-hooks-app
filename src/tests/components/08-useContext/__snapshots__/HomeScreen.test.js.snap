// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test on <HomeScreen/> should rendered right  1`] = `
<div>
  <h1>
    Home Screen
  </h1>
  <hr />
  <pre>
    {
   "name": "papaleta",
   "email": "pp@mail.com"
}
  </pre>
</div>
`;
